### ISSUE TITLE
[Bug][Radio Button] Regression : impossible to select a radio button item in group

### ISSUE THREAD

**Issue by pya35** [Orange-OpenSource/ouds-flutter#615]
### Prerequisites

- [x] I have [searched the backlog](https://github.com/Orange-OpenSource/ouds-flutter/issues) for duplicate or closed feature requests

### Your test device

iPhone 

### OS Version

26

### App version

1.0.0 (649)

### Describe the issue

Radio button items are not selectable anymore with Voice Over or keyboard

<img width="516" height="230" alt="" src="https://github.com/user-attachments/assets/5b63cbde-bdd6-4553-88b4-4625d261cacf" />

### Expected Behavior

Each radio button should be selectable, as done before
